views/tenants/new.erb

```diff
@@ -1 +1,8 @@
-form for adding a new tenant
+<h2>Add New Tenent for <%= @apartment.id %></h2>
+
+<form method="put" action="/tenents/create">
```

Comment:
One small suggestion:
1) The customary route for creating would be PUT "/tenants", no need for the "/create" at the end.